ui/src/layout/Layout.tsx

```diff
@@ -58,10 +61,17 @@ class Layout extends React.Component<WithStyles<'content'> & Stores<'currentUser
                 this.version = resp.data.version;
             });
         }
+
+        const localStorageTheme = window.localStorage.getItem(localStorageThemeKey);
+        if (localStorageTheme) {
+            this.currentTheme = localStorageTheme as ThemeKey;
```

Comment:
This type assertion could produce problems in the future. For example if the 'light' theme would be removed from gotify then some users may still have "light" in their local storage. This would result in a `undefined` theme, because 'light' isn't present anymore:
https://github.com/gotify/server/blob/1f0f7d471517d527677009beacc6875e2c1bebc2/ui/src/layout/Layout.tsx#L84

Anyway, this could be fixed with a [User-Defined Type Guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards). It could look like this:
```typescript
const isThemeKey = (value: any): value is ThemeKey => {
    return value === 'light' || value === 'dark';
}
```
and be used like this:
```typescript
const localStorageTheme = window.localStorage.getItem(localStorageThemeKey);
if (isThemeKey(localStorageTheme)) {
    this.currentTheme = localStorageTheme;
} else {
    window.localStorage.setItem(localStorageThemeKey, this.currentTheme);
}
```